src/common/security/context.go

```diff
@@ -27,6 +27,8 @@ type Context interface {
 	GetUsername() string
 	// IsSysAdmin returns whether the user is system admin
 	IsSysAdmin() bool
+	// PolicyCheck returns whether the middlerwares apply to the request
```

Comment:
I don't think we need this, we can modify the Can func for robot security context and make it return true for a special action.

Reply:
updated.